auth/user_mgt_test.go

```diff
@@ -11,7 +11,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
```

Comment:
A new line here is preferred. Please add it back, and I will merge.

Reply:
done